sysutils/lima/files/patch-Makefile.diff

```diff
@@ -9,3 +9,12 @@
  VERSION_TRIMMED := $(VERSION:v%=%)
  
  GO_BUILD := $(GO) build -ldflags="-s -w -X $(PACKAGE)/pkg/version.Version=$(VERSION)" -tags "$(GO_BUILDTAGS)"
+@@ -209,7 +209,7 @@
+ install: uninstall
+ 	mkdir -p "$(DEST)"
+ 	# Use tar rather than cp, for better symlink handling
+-	( cd _output && tar c * | tar Cxv "$(DEST)" )
++	( cd _output && tar c * | $(TAR) -xv --no-same-owner -C "$(DEST)" )
```

Comment:
`tar c *` should probably be `$(TAR) c *` if you’re changing it on the right side of the pipe.

Reply:
Thanks, fixed!